tests/share/normalize/factories.py

```diff
@@ -363,7 +363,7 @@ def _parse(self, *args, **kwargs):
 
 
 def _params(seed=None, id=None, type=None, **kwargs):
-    string_id = '_:_' + str(id)
+    string_id = id if isinstance(id, str) else '_:_' + str(id)
```

Comment:
For submitting changes with a non-blank SHARE ID (like to add an identifier to a specific work, triggering a merge). Something like `_:_461C1-98C-F8E` doesn't work.